clients/v1/clients.proto

```diff
@@ -168,6 +180,26 @@ message UpdateUserMetadataResponse {
   UserServiceMetadata metadata = 1;
 }
 
+message CreateUserRequest {
+  // The email address for the new user.
+  string email = 1;
+  // The name for the new user
+  string name = 2;
+}
+
+message CreateUserResponse {
+  // The newly created user.
+  User user = 1;
+}
+
+message DeleteUserRequest {
+  // The email address of the user to delete.
+  string email = 1;
```

Comment:
The current management api uses the external id which would be preferable
```suggestion
  // The user's external ID.
  string id = 1;
```

Reply:
do you think it's better to stick to what the current implementation is doing? 

if so, we'll have to do a call to get the users ID, then call deleteUser

Reply:
That's okay and is how the current github actions script works.
If you look at the other RPCs which peform actions on a user they all use the user id.

Email serves as an entrypoint into SAMS but from there the SAMS ID is the primary identifier of a user

Reply:
sounds good i'll update the rest of the PRs to use SAMS id then!